dashboard/src/app/index.module.ts

```diff
@@ -65,18 +73,18 @@ function keycloakLoad(keycloakSettings: any) {
   return new Promise((resolve: IResolveFn<any>, reject: IRejectFn<any>) => {
     const script = document.createElement('script');
     script.async = true;
-    script.src = keycloakSettings['che.keycloak.auth_server_url'] + '/js/keycloak.js';
+    script.src = keycloakSettings['che.keycloak.js_adapter_url'];
     script.addEventListener('load', resolve);
     script.addEventListener('error', () => reject('Error loading script.'));
     script.addEventListener('abort', () => reject('Script loading aborted.'));
     document.head.appendChild(script);
   });
 }
-function keycloakInit(keycloakConfig: any) {
+function keycloakInit(keycloakConfig: any, theUseNonce: boolean) {
```

Comment:
useNonce instead of theUseNonce ?

Reply:
it was to avoid a duplicate with the `useNonce` named parameter of the `keycloak.init()` method.